SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/OwnershipLockingPolicy.cpp

```diff
@@ -60,13 +60,13 @@ bool UOwnershipLockingPolicy::ReleaseLock(const ActorLockToken Token)
 	const LockNameAndActor* NameAndActor = TokenToNameAndActor.Find(Token);
 	if (NameAndActor == nullptr)
 	{
-		UE_LOG(LogOwnershipLockingPolicy, Error, TEXT("Called ReleaseLock for unidentified Actor lock token. Token: %d."), Token);
+		UE_LOG(LogOwnershipLockingPolicy, Error, TEXT("Called ReleaseLock for unidentified Actor lock token. Token: %lld."), Token);
```

Comment:
If you have a timed lock (which has been talked about a bit for this) then this will throw up false errors if you release a lock after the timer has expired. I realised this is just changing the token type but it's probably going to come up as an error.

Reply:
Agreed, happy to discuss in a followup ticket / PR